helm/charts/determined/templates/master-config.yaml

```diff
@@ -193,6 +193,9 @@ stringData:
 
     {{ if .Values.taskContainerDefaults -}}
     task_container_defaults:
+      {{- if .Values.taskContainerDefaults.startupHook }}
+      startup_hook: {{ .Values.taskContainerDefaults.startupHook | quote}}
```

Comment:
works fine?
```
 and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
[2024-04-10T16:04:10.309056Z] e8046e80 || + test -f /run/determined/dynamic-tcd-startup-hook.sh
[2024-04-10T16:04:10.309104Z] e8046e80 || + source /run/determined/dynamic-tcd-startup-hook.sh
[2024-04-10T16:04:10.309121Z] e8046e80 || ++ echo 'Hello, world from master startup!'
[2024-04-10T16:04:10.309175Z] e8046e80 || Hello, world from master startup!
[2024-04-10T16:04:10.309277Z] e8046e80 || ++ ls /etc/issue
[2024-04-10T16:04:10.309933Z] e8046e80 || /etc/issue
[2024-04-10T16:04:10.310102Z] e8046e80 || ++ date
```

for 

```
  # Configure an optional inline script that will be executed as part of the task setup process.
  startupHook: |
    echo "Hello, world from master startup!"
    ls /etc/*issue
    date

```

Reply:
although it missed the "*" from the ls line let me put that in separate quotes

Reply:
ok cool, as long as you've tested it that's fine with me. let @BOsterbuhr take a look because it's different than what he wrote so i'm curious what he thinks.

Reply:
👍🏼 
```
[2024-04-10T16:07:15.253015Z] 1e8c256b || ++ ls '/etc/issu*'
[2024-04-10T16:07:15.253103Z] 1e8c256b || Hello, world from master startup!
[2024-04-10T16:07:15.253610Z] 1e8c256b || ls: cannot access '/etc/issu*': No such file or directory
[2024-04-10T16:07:15.485994Z] 1e8c256b || ERROR: crashed: resources failed with non-zero exit code:  (ex
```

Reply:
Yeah the toYaml was if it was 
```
startupHook:
    cmd: ["echo", "Hello World"]
```
but since it is just a string we just have to quote it and we are good.

Looks good to me @hamidzr